indy-besu/vdr/Cargo.toml

```diff
@@ -21,6 +21,9 @@ basic_signer = ["secp256k1"]
 migration = []
 
 [dependencies]
+tokio = "1.34.0"
+futures-core = "0.3"
+futures-util = "0.3"
```

Comment:
Are these dependencies (`futures-core` and `futures-util` ) really need to be listed here?

Reply:
No. Removed